Makefile

```diff
@@ -0,0 +1,95 @@
+
+BUILD_DIR=build
+OUTPUT_DIR=$(BUILD_DIR)/_site
+AWESTRUCT_CONFIG=--source-dir=content --output-dir=$(OUTPUT_DIR)
+ASSETS_DIR=$(OUTPUT_DIR)/assets/bower
+VERSION=$(BUILD_NUMBER)-$(shell git rev-parse --short HEAD)
+
+# Generate everything
+all: fetch generate pdfs archive
+prepare: fetch depends assets
+
+# Run a local dev server on localhost:4242
+run: warning depends-ruby assets
+	LISTEN=true ./scripts/ruby bundle exec awestruct --bind 0.0.0.0 --dev $(AWESTRUCT_CONFIG)
```

Comment:
Since this is executing in the container, it must bind inside that container to 0.0.0.0 so the Docker daemon can map the port properly